docs/api.rst

```diff
@@ -485,6 +485,55 @@ Possible Responses
 | 500 Server Error | Internal Raiden node error|
 +------------------+---------------------------+
 
+
+Querying token specific events
+------------------------------
+
+You can query for all new channels opened for a token by making a ``GET`` request to the following endpoint. ``GET /api/<version>/events/token/<token_address>``
```

Comment:
Same reason I used it for channel. Because afterwards the token address follows. As a user I would expect to be able to provide many token addresses if I saw plurar here.